doc/Overview/Maui/MauiOverview.md

```diff
@@ -1,88 +1,261 @@
+
+![Select startup type](Assets/GettingStarted01-StartupType.png)
+
+.NET MAUI Embedding is only supported on iOS, Android, MacCatalyst and Windows. If you have any other platforms selected, the .NET MAUI Embedding feature will be disabled. Select the Platforms section and make sure only iOS, Android, MacCatalyst and Windows is selected.  
+
+![Select platforms](Assets/GettingStarted02-Platforms.png)
+
+Select the Features section and include the MAUI Embedding feature, before clicking the Create button to complete the creation of the application.  
+
+![Select feature - .NET MAUI Embedding](Assets/GettingStarted03-MAUIEmbedding.png)
+
+In addition to the typical Uno Platform application structure, there is a .NET MAUI class library (eg SimpleMauiApp.MauiControls). The MauiControls class library is where you add references to third party .NET MAUI control libraries, add any services or call registration/initialization methods on the third party libraries, and of course define the .NET MAUI controls that you want to display with the application. 
+
+![Project structure](Assets/GettingStarted04-ProjectStructure.png)
+
+The application can be run by setting the desired platform project (Mobile or Windows) to be the startup project and then pressing F5 or clicking the Run button. The default layout shows a text, "Hello Uno Platform", which is an Uno TextBlock, followed by an Image, two TextBlock and a Button, which are all .NET MAUI controls.
+
+![Running on Windows](Assets/GettingStarted05-RunningOnWindows.png)
+
+These steps can also be achieved via the command lines by invoking the following commands 
 
-```cs
-protected override void OnLaunched(LaunchActivatedEventArgs args)
-{
-    this.UseMauiEmbedding();
-}
+```
+dotnet new install uno.templates
+dotnet new unoapp -preset blank -maui -o SimpleMauiApp
 ```
 
-Similarly if you are using Uno.Extensions.Hosting for Dependency Injection you can call the `UseMauiEmbedding()` extension off of the `IApplicationBuilder` like:
+## Get Started - Existing Uno Application
 
-```cs
-protected override void OnLaunched(LaunchActivatedEventArgs args)
-{
-    var builder = this.CreateBuilder(args)
-        .UseMauiEmbedding();
-}
+.NET MAUI Embedding can be added to any existing Uno application via the following steps. The .NET MAUi Embedding feature is only supported in Uno applications that target iOS, Android, MacCatalyst and Windows, as shown in the Platforms folder in the following solution structure. These instructions are appropriate for an Uno application targeting .NET 7.
+
+![Starting solution structure](Assets/GettingStartedExisting01-StartProjectStructure.png)
+
+1.  **Uno.Extensions.Maui.WinUI**  
+Add a reference in the existing class library to [Uno.Extensions.Maui.WinUI](https://www.nuget.org/packages/Uno.Extensions.Maui.WinUI). It's not necessary to add this package to other projects.  
+
+![Adding reference to Uno.Extensions.Maui.WinUI](Assets/GettingStartedExisting02-AddingReference.png)  
```

Comment:
Lack of indentation will break numbering. Applies here and in other cases below (both code snippets & images)